src_code/src/routes/AuthenticationRoutes.js

```diff
@@ -14,17 +14,60 @@ router.post('/sign-in', async(request, response) => {
         /** If there is no request body, then we return the request body is empty */
         if(!request.body || Object.keys(request.body).length === 0) {
             /** Return request body is empty */
-            return responseBuider.MissingContent(response, "RB");
+            return responseBuilder.MissingContent(response, "RB");
         } 
         /** If request body is exist, then we perform sign in based on request body */
         return await authenticationServices.SignIn(response, request.body);
     }catch(error){
         /** Logging unexpected error. help for debug */
         console.log("ERROR: There is an error while signing-in: ", error);
         /** Response error message to the client */
-        return responseBuider.ServerError(response, "There is an error while sign in.");
+        return responseBuilder.ServerError(response, "There is an error while sign in.");
     }
 })
 
-module.exports = router;
+/**
+ *  POST/SIGN UP 
+ *  URL => /api/authentication/sign-up
+ */
+router.post("/sign-up", async(req, res) => {
+    try{
+         /** If there is no request body, return missing request body */
+        if (!req.body || Object.keys(req.body).length === 0) {
+            return responseBuilder.MissingContent(res, "RB");
+        }
+        /** return response to the client */
+        return await authenticationServices.SignUp(res, req.body);
+    }catch(error){
+        /** If error, return log and response with server error message */
+        console.log("ERROR: There is an error occur while signing up: ", error);
+        return responseBuilder.ServerError(
+        res,
+        "Sorry, an error occur while signing up."
+        );
+    }
+})
 
+/**
+ *  POST/SEND VERIFICATION CODE
+ *  URL => /api/authentication/send-verification-code
+ */
+router.post("/send-verification-code", async(req, res) => {
+    try{
+         /** If there is no request body, return missing request body */
+        if (!req.body || Object.keys(req.body).length === 0) {
+            return responseBuilder.MissingContent(res, "RB");
+        }
+        /** return response to the client */
+        return await authenticationServices.SendVerificationCode(res, req.body);
```

Comment:
nit: same kind of redundant `await` here

Reply:
Await is not redundant. It should be used with resolved Promise. Async function returns promise but javascript does not ensure that promist is successfully resolve. I will update it.